ui/raidboss/data/02-arr/raid/t11.js

```diff
@@ -110,10 +96,7 @@
       regexCn: Regexes.hasHP({ name: '卡利亚', hp: '60', capture: false }),
       regexKo: Regexes.hasHP({ name: '칼리야', hp: '60', capture: false }),
       sound: 'Long',
-      infoText: {
-        en: 'Out of Middle',
-        fr: 'En dehors du centre',
-      },
+      response: Responses.getOut(),
```

Comment:
This is not the same.  Get out is ambiguous and usually means away from the boss.  This means don't stand in the center of the arena.